lib/chore/queues/filesystem/publisher.rb

```diff
@@ -24,6 +24,9 @@ def publish(queue_name,job)
               if f.flock(File::LOCK_EX | File::LOCK_NB) && f.size == 0
                 begin
                   f.write(job.to_json)
+                rescue StandardError => e
+                  Chore.logger.error "#{e.class.name}: #{e.message}. Could not write #{job[:class]} job to #{queue_name} file.\n#{e.stacktrace}"
+                  raise e
```

Comment:
Also, I don't know if we want to raise here—that would be very new behavior, and I'm not sure if raising in the publishing code is always safe.

That said, I think we mostly use the filesystem queue in dev/tapinabox environments.